libraries/AP_AHRS/AP_AHRS_NavEKF.h

```diff
@@ -115,6 +115,9 @@ class AP_AHRS_NavEKF : public AP_AHRS_DCM {
     }
 #endif
 
+    // return the quaternion defining the rotation from NED to XYZ (body) axes
+    bool get_quaternion(Quaternion &quat) const override;
```

Comment:
`WARN_IF_UNUSED` ?

Reply:
Thanks added.  Note that the other get_secondary_attitude(), get_secondary_quaternion(), etc don't have this but two (or three or four!) wrongs don't make a right!